src/platform/testing/unit/mocha-setup.js

```diff
@@ -26,6 +26,12 @@ export default function setupJSDom() {
   //   throw new Error('Refusing to override existing document and window.');
   // }
 
+  // Prevent warnings from displaying
+  /* eslint-disable no-console, func-names */
+  console.error = function() {};
+  console.warn = function() {};
```

Comment:
Does declaring these as arrow functions avoid the linter from complaining about `func-names`?

Reply:
Can't remember why I didn't have them as arrow functions in the first place. I just converted them.